<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="text_to_image_button">
         <property name="text">
          <string>Txt2Img</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="make_tilable_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Make Tilable</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="gfpgan_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>GFPGAN</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="inpaint_button">
         <property name="text">
          <string>Inpaint</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="image_to_image_button">
         <property name="text">
          <string>Img2Img</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="upscale_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Upscale</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="settings_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="quick_mask_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Creates transparency mask for current layer</string>
         </property>
         <property name="text">
          <string>Quick Mask</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
